client/src/App/PaginatedBacklog/Backlog/Backlog.jsx

```diff
@@ -0,0 +1,23 @@
+import "./Backlog.css";
+export function BacklogTaskList({ backlogTasks }) {
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Task name</th>
+        </tr>
+      </thead>
+      <tbody>
+        {backlogTasks.map((backlogTask, index) => {
+          return (
+            <tr key={index}>
```

Comment:
The key is only needed on the outer element, in this case `tr`. But you want to avoid using `index` for this, you can read the [React docs](https://react.dev/learn/rendering-lists#why-does-react-need-keys) again why that is. `backlogTask.id` is far more suitable 🙂